omod/src/test/java/org/openmrs/module/idgen/rest/resource/AutoGenerationOptionResourceTest.java

```diff
@@ -0,0 +1,78 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.idgen.rest.resource;
+
+import org.openmrs.module.idgen.AutoGenerationOption;
+import org.openmrs.api.context.Context;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.module.idgen.service.IdentifierSourceService;
+import org.openmrs.module.idgen.rest.resource.AutoGenerationOptionResource;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.Hyperlink;
+//import org.openmrs.module.idgen.web.RestTestConstants;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+
+public class AutoGenerationOptionResourceTest extends BaseDelegatingResourceTest<AutoGenerationOptionResource, AutoGenerationOption> {
+	
+	@Before
+	public void setUp() throws Exception {
+    executeDataSet("org/openmrs/module/idgen/include/TestData.xml");
```

Comment:
Did you intentionally indent the above like so?

Reply:
My error. I'll correct it now.